### DISCUSSION TITLE
Probably out of order signage point IDs

### DISCUSSION THREAD

**Discussion by marramgg** [martomi/chiadog#143] [Q&A]
Hello, this might come around frequently however I was unable to find an answer in the Discussions or Issues of this repo.

Chiadog has constant logs of `Probably out of order signage point IDs X and Y with timestamps A and B (calculate_skipped_signage_points.py:22)`

I've read [this post](https://github.com/martomi/chiadog/discussions/85#discussioncomment-701000) where @martomi replies to a question about farming events however it doesn't clarify if the out of order signage points is actually something that could cause a proof from being found in one of my plots.

Do you have any suggestions on how to prevent out of order signage points? I don't think it's for lack of resources so I'm assuming (details about the system below) so my next guess is that it's something related to Chia itself and this is not a harmful or actionable notification. Does this make sense? Thanks!

Info about the system: Ryzen 9 3950X, 64GB ECC, OS on NVMe and plotting on NVMEs, all HDDs with plots locally. Local network and internet both symmetric gbit connection.

**Comment by martomi** [comment]
Short answer: It's expected when network forks happen. The newest version hides these logs in DEBUG level.

Long answer: See this commit and the surrounding context https://github.com/martomi/chiadog/commit/e3591116ebd9ebb1845162346098e286a95f7002
